packages/common/src/ui/layout/tables/DataTable.tsx

```diff
@@ -103,6 +103,7 @@ export const DataTable = <T extends DomainObject>({
           {data.map((row, idx) => {
             return (
               <DataRow
+                rows={data}
```

Comment:
This is a little bit of left over work from trying a different way which just had the `accessor` passed all of the rows of the table.. it didn't really work out (as the types got all messed up as we still had to reduce the rows and create some grouping/aggregate rows and it just got a bit messy..). I've left it because it means the param for the `accessor` fn is currently `{rowData, rows}`. Though the `rows` isn't used anywhere as of right now, it might be used.. but the main thing is making this param into an object where we might in the future add more values `(i.e. maybe you want access to the column object.. 🤷 ) so thought to leave it in.